### PR TITLE
ESLint

### PR DESCRIPTION
**Pull request by chrismiceli**
Stylecop dat crap

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -1,73 +1,74 @@
...
+    wellplayed: 'Well played.',
+    whatagame: 'What a Game!'
 };
 
 exports.getCommand = function (command) {
   // Break out the command so we can parse it, and initialize the finalCommand to invalid.
-  var commands = command.split(" ");
-  var finalCommand = { type: "invalid", text: "", times: 0 };
+    const commands = command.split(' ');
+    const finalCommand = { type: 'invalid', text: '', times: 0 };
```
Comment (KevinGAleman) [inline commands.js:52]: This var changes; probably shouldn't be a const.

**Review by KevinGAleman — APPROVED** [review]
One little fix then it's golden. 👍

File: commands.js (hunk 1 above)
Comment (chrismiceli) [reply commands.js:52]: The use of const here is valid, unless slack is running an older version of node that doesn't support const. You can set properties of a const, just not reassign the variable.

**Comment by KevinGAleman** [comment]
Welp. Sounds good. Ship it.

**Comment by chrismiceli** [comment]
I don't think I can merge. Do you have the appropriate permissions?

**Pull request merged**